### ISSUE TITLE
InvalidQueryException when inserting rows with udt type column set to null

### ISSUE THREAD

**Issue by dyangrev** [Stratio/cassandra-lucene-index#149]
Hey we're seeing query errors when inserting rows with udt type column set to null if the table has lucene index, here're the detail: 
cqlsh> CREATE KEYSPACE test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
cqlsh> use test;
cqlsh:test> CREATE TYPE blah_t (a text);
cqlsh:test> CREATE TABLE blah (foo text, bar frozen <blah_t>, primary key (foo));
cqlsh:test> CREATE CUSTOM INDEX blah_idx on blah() using 'com.stratio.cassandra.lucene.Index' with options = {'refresh_seconds': '1', 'schema': '{fields: {"bar.a": {type:"string"}}}'};
cqlsh:test> INSERT INTO blah (foo, bar) values ('a', 'a');
cqlsh:test> INSERT INTO blah (foo, bar) values ('b', null);
**InvalidRequest: code=2200 [Invalid query] message="0"**
cqlsh:test> delete bar from blah where foo = 'a';
**InvalidRequest: code=2200 [Invalid query] message="0"**
cqlsh:test> DROP INDEX blah_idx;
cqlsh:test> delete bar from blah where foo = 'b';
cqlsh:test> delete bar from blah where foo = 'a';
cqlsh:test> INSERT INTO blah (foo, bar) values ('c', null);   

Noticed that this only happen to columns with frozen <udt>. We tried both version 3.0.3 and 3.0.6.

**Comment by adelapena** [comment]
Hi @dyangrev ,

Yes, this is a bug affecting frozen data types: UDTs, tuples and frozen collections. 

I have [just fixed it](https://github.com/Stratio/cassandra-lucene-index/commit/3ef86243464ae39e84a56e1c42ec46d85016ef63) on top of [branch 3.0.6](https://github.com/Stratio/cassandra-lucene-index/tree/branch-3.0.6). The fix will be available in next release, 3.0.6.2, probably on Monday.

Thanks for reporting.

**Comment by jrglee** [comment]
Hi @adelapena, any chance to apply this fix to branches before 3.0.6? Od do you recommend to upgrade cassandra itself to 3.0.6?

**Comment by adelapena** [comment]
Hi @jrglee, 

As we promised we have just released [3.0.6.2](https://github.com/Stratio/cassandra-lucene-index/releases/tag/3.0.6.2) with the hotfix. 

Cassandra 3.0.6 is the latest and most stable 3.0.x release, [containing several fixes and improvements](https://github.com/apache/cassandra/blob/cassandra-3.0.6/CHANGES.txt), so we think it is a good idea to upgrade to Apache Cassandra 3.0.6 with Lucene's index 3.0.6.2. At some point we will port back some of the fixes to past versions, but our main priority are the latest 2.x and 3.0.x releases, specially 3.0.x, just because they are the Cassandra recommended versions.

Again, thanks for reporting the bug.
